apps/backend/src/auth/auth.controller.ts

```diff
@@ -0,0 +1,29 @@
+import {
+  Body,
+  Controller,
+  Post,
+  UseGuards,
+  ValidationPipe,
+} from '@nestjs/common';
+import { AuthService } from './auth.service';
+import { Public } from './decorator/public.decorator';
+import { AuthGuard } from './guard/auth.guards';
+import { LoginDataDto } from './dto/login.dto';
+
+@Controller('auth')
+@UseGuards(AuthGuard)
+export class AuthController {
+  constructor(private authService: AuthService) {}
+
+  @Post('/login-parish')
+  @Public()
+  loginParish(@Body(ValidationPipe) input: LoginDataDto) {
```

Comment:
Rather than passing it everywhere as you do, add the `app.useGlobalPipes(new ValidationPipe())` expression to your `main.ts` file as described at https://docs.nestjs.com/techniques/validation#auto-validation